### PR TITLE
docs: add LRNWebcomponents to component libraries

### PR DESCRIPTION
**Pull request by btopro**
Added link to LRNWebcomponents which contains HAX and 100s of other openly available, primarily Lit based elements.

### CODE REVIEW

**Comment by Latanya541** [comment]
Open

File: docs/_data/componentLibraries.js [hunk 1]
```diff
@@ -101,6 +101,12 @@ const componentLibraries = [
     description:
       'Lion web components is a set of highly performant, accessible and flexible Web Components. They provide an unopinionated, white label layer that can be extended to your own layer of components.',
   },
+  {
+    name: 'LRNWebcomponents',
+    url: 'https://github.com/elmsln/lrnwebcomponents',
```
Comment (Westbrook) [inline docs/_data/componentLibraries.js:106]: Do you have a public URL that more directly points to the components in the library, and possibly how to consume them?

Comment (btopro) [reply docs/_data/componentLibraries.js:106]: https://webcomponents.psu.edu/ isn't comprehensive but has a lot of them, you mean like that?

Comment (Westbrook) [reply docs/_data/componentLibraries.js:106]: That definitely gave devs a bette rides of what they might be able to get from the library, yes

**Review by web-padawan — APPROVED** [review]

**Pull request merged**